language-design.md

```diff
@@ -38,16 +128,50 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+##### A well-designed language:
+
+* has a system or pattern for growth so that it can evolve with the needs of its users.
+* stays true to a specific user, purpose, or philosophy so that development is directed and the language doesn't become obscure because it's trying to be too many things at once.
```

Comment:
While this may be true for a DSL, shouldn't a GPL try to do a lot of things well?  Why else would popular languages like Python, Java, and C++ have **massive* standard libraries?